### DISCUSSION TITLE
自动获取复制的链接后会造成解析或下载时的问题

### DISCUSSION THREAD

**Discussion by canyuehen** [xisuo67/XHS-Spider#84] [Polls]
不知道大家有没有这个问题，
1、我复制了主页链接进行解析，最小化了软件。
2、在解析过程中我看到了一个视频，就复制了链接用小程序下载，但是软件就会自动获取复制的链接并且确认，之前解析了的就会消失，要重新操作第一步。
整体来说就是在解析和下载过程中不能复制链接

**Comment by xisuo67** [comment]
是的，你说的场景会有，我觉得程序既想享受快捷便利，又需要程序自动区分使用场景的情况很难办到
在程序解析过程中，尽量避免复制的功能，或者有更好的方法，欢迎提供解决思路

**Comment by xisuo67** [comment]
将问题转入讨论贴中，有更好的解决方案欢迎大家一起来讨论

**Comment by xisuo67** [comment]
这个问题会在2.0版本中修复，增加是否监听剪切板参数，避免误操作，同时增强剪切板监听操作，同时按下两次Ctrl+C才会触发监听剪切板事件！
